app/src/sharedTest/java/org/oppia/app/utility/RatioExtensionsTest.kt

```diff
@@ -34,6 +30,16 @@ class RatioExtensionsTest {
     ).isEqualTo("1 to 2 to 3")
   }
 
+  @Test
+  fun testRatio_ratioLengthTwoF_returnAccessibleRatioString() {
```

Comment:
Typo in name--please fix.

Also, 'returns'. Ditto above.

Reply:
done